python/ray/tune/trial_executor.py

```diff
@@ -8,6 +10,16 @@
 logger = logging.getLogger(__name__)
 
 
+def get_warn_threshold(autoscaler_enabled: bool) -> float:
```

Comment:
can we make this private actually? `_get_warn_threshold`

Reply:
good point. made it private.